### PR TITLE
JSON Method for Structured Provider

### PR DESCRIPTION
**Pull request by wikkiewikkie**
@lk-geimfari As requested

### CODE REVIEW

File: elizabeth/core/elizabeth.py [hunk 1]
```diff
@@ -474,6 +475,36 @@ def html_attribute_value(self, tag, attribute):
...
+        :param _recursive: When used recursively, will return a \
+        python object instead of JSON string.
+        :type _recursive: bool
+        :return: JSON
+        :rtype: str
+        """
+        root = choice([list, dict])()  # choose root element type
+        for _ in range(items):
+            if max_depth > 0:
+                data = choice([self.text.sentence(),
```
Comment (lk-geimfari) [inline elizabeth/core/elizabeth.py]: The data should be from `Personal()` i.e username, name, age, email etc.

Comment (lk-geimfari) [inline elizabeth/core/elizabeth.py]: Like in example below:

```json
{
  "users": [
    {
      "username": "",
      "full_name": "",
      "email": "",
      "age": ""
    },
    {
      "username": "",
      "full_name": "",
      "email": "",
      "age": ""
    },
    {
      "username": "",
      "full_name": "",
      "email": "",
      "age": ""
    }
  ]
}
```

**Comment by lk-geimfari** [comment]
@wikkiewikkie Look above, please.

**Comment by wikkiewikkie** [comment]
@lk-geimfari I see what you are saying and wonder if these methods make sense in a separate Structured provider.  It seems like it would make more sense to have a json method on the Personal provider that would return something like the example you posted above.  Then other providers could also implement json methods.

For example, Hardware.json() would return:

```json
{ 
    "computers": [
        {
        "resolution": "1280x720",
        "cpu": "Intel® Core i7",
        "ram_type": "DDR3",
        "graphics": "Intel® Iris™ Pro Graphics 6200"
        },
        {
        "resolution": "1920x1080",
        "cpu": "Intel® Core i5",
        "ram_type": "DDR5",
        "graphics": "Intel® Iris™ Pro Graphics 5000"
        }
    ]
}
```

What do you think?

**Comment by lk-geimfari** [comment]
@wikkiewikkie It's good idea. We can realize it like in example below

```python
>>> from elizabeth import Structured
>>> structured = Structured('en')

>>> structured.json(provider='hardware')
```
Output:
```json
{ 
    "computers": [
        {
        "resolution": "1280x720",
        "cpu": "Intel® Core i7",
        "ram_type": "DDR3",
        "graphics": "Intel® Iris™ Pro Graphics 6200"
        },
        {
        "resolution": "1920x1080",
        "cpu": "Intel® Core i5",
        "ram_type": "DDR5",
        "graphics": "Intel® Iris™ Pro Graphics 5000"
        }
    ]
}
```

```python
>>> structured.json(provider='personal')
```
Output:
```json
{
  "users": [
    {
      "username": "",
      "full_name": "",
      "email": "",
      "age": ""
    },
    {
      "username": "",
      "full_name": "",
      "email": "",
      "age": ""
    },
    {
      "username": "",
      "full_name": "",
      "email": "",
      "age": ""
    }
  ]
}
```

**Comment by lk-geimfari** [comment]
@w

**Comment by wikkiewikkie** [comment]
@lk-geimfari Please take a look at my revisions per our discussion above when you have a chance.  Thanks.

**Comment by lk-geimfari** [comment]
@wikkiewikkie Much better. Thank you!

**Pull request merged**